aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -301,7 +301,7 @@ webBuilder.CaptureStartupErrors(true);
 When enabled, or when the environment is `Development`, the app captures detailed errors.
 
 **Key**: `detailedErrors`  
-**Type**: `bool` (`true` or `1`)  
+**Type**: `bool` (`true`/`1` or `false`/`0`)  
 **Default**: `false`  
 **Environment variable**: `<PREFIX_>_DETAILEDERRORS`
```

Comment:
There's a few cases of this `<PREFIX_>_` double `_` in here. Do you want to fix them while you're in here, or shall I patch them up in a separate PR?

Reply:
I'll fix them ... and I think we've (or I've) been settling more on braces for these.